### DISCUSSION TITLE
Unable to specify Transform during database table import/export

### DISCUSSION THREAD

**Discussion by battmush** [dbeaver/dbeaver#36659] [Q&A] [answered]
### Description

When selecting that 'table' option for import/export data transfer for copying a table, the Transform column cannot be interacted with; i.e. there is no drop-down t provide a transform method.  This is true regardless this is approached as an Import or an Export, or whether the table is existing or new (to-be-created).  I've tried several different destination column types without change.  The source database is iSeries AS400 DB2, destination is Postgres (16+).

### DBeaver Version

Version 24.2.5.202411171748

### Operating System

Windows 11 and RedHat EL8

### Database and driver

Source: DB2 iSeries JTOpen: net.sf.jt400:jt400:RELEASE: https://github.com/IBM/JTOpen/releases/tag/v20.0.7

Destination: Postgres JDBC: org.postgresql:postgresql:RELEASE [42.7.2]

### Steps to reproduce

1. Open connection to source iSeries database
2. Open connection to destination Postgres database
3. Select the table to export/import/copy, 
4. Right-click > Export
5. Select Database | Database table(s)
6. Under Table Mapping, attempt to interact with the Transform column for the table (and column)
![Clipboard_11-25-2024_01](https://github.com/user-attachments/assets/da2c9c92-f34f-4ffe-a293-4fc42b36a35d)

### Additional context

I've read the documentation, and it gives zero indication of why this may be the case.

https://github.com/dbeaver/dbeaver/wiki/Data-migration#transform-column-values

**Comment by ShadelessFox** [comment, marked as answer]
Hello @battmush,

As the article shows, column transforms should be specified in the data transfer wizard itself:

![image](https://github.com/user-attachments/assets/6ea3cf73-b29c-4729-a8b3-621eaa20f592)

However, you can't modify them or change the mapping type in the popup. This is intentional.

Please let me know if you have any other questions.
